src/wpt/dom/events-test.ts

```diff
@@ -45,9 +45,7 @@ export default {
   'EventTarget-addEventListener.any.js': {},
   'EventTarget-constructible.any.js': {
     comment: 'Should be null, not EventTarget',
-    expectedFailures: [
```

Comment:
Shouldn't there be a change relating to `isTrusted` here?

Reply:
I thought so too but it passed...